src/core/linq/observable/fromevent.js

```diff
@@ -77,15 +77,16 @@
    * Creates an observable sequence by adding an event listener to the matching DOMElement or each item in the NodeList.
    * @param {Object} element The DOMElement or NodeList to attach a listener.
    * @param {String} eventName The event name to attach the observable sequence.
+   * @param {Object} eventListenerOptions An object describing EventListenerOptions
    * @param {Function} [selector] A selector which takes the arguments from the event handler to produce a single item to yield on next.
    * @returns {Observable} An observable sequence of events from the specified element and the specified event.
    */
-  Observable.fromEvent = function (element, eventName, selector) {
+  Observable.fromEvent = function (element, eventName, eventListenerOptions, selector) {
     // Node.js specific
     if (element.addListener) {
       return fromEventPattern(
-        function (h) { element.addListener(eventName, h); },
-        function (h) { element.removeListener(eventName, h); },
+        function (h) { element.addListener(eventName, h, eventListenerOptions); },
+        function (h) { element.removeListener(eventName, h, eventListenerOptions); },
```

Comment:
I don't understand, you said this is being added to the Chrome 51 spec, but the only places you have changed are non-native adapter code for Node and several popular frameworks.